app/routes/admin/email/components/EmailUsers.tsx

```diff
@@ -35,15 +42,16 @@ const EmailUsers = () => {
     selectTransformedEmailUsers(state, { pagination }),
   );
   const fetching = useAppSelector((state) => state.emailUsers.fetching);
-  const committees = useAppSelector((state) =>
-    selectGroupsByType(state, GroupType.Committee),
-  );
-  const boards = useAppSelector((state) =>
-    selectGroupsByType(state, GroupType.Board),
-  );
-  const ordained = useAppSelector((state) =>
-    selectGroupsByType(state, GroupType.Ordained),
-  );
+
+  const relevantGroups = [
+    ...useAppSelector((state) =>
+      selectGroupsByType(state, GroupType.Committee),
+    ),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.Board)),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.Ordained)),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.SubGroup)),
+  ];
```

Comment:
Wouldn't it be better to create a new selector for this?

Reply:
You do have a very good point

Reply:
Do you have any comments on the changes to the selector @ivarnakken?